src/atoms/story-elements/text/text.tsx

```diff
@@ -105,10 +79,11 @@ export const DefaultText = ({ element, config, story }: StoryElementProps) => {
   if (element.subtype === "cta") {
     return <CtaText element={element} dangerouslySetInnerHTML={{ __html: text }} />;
   }
-  if (storyTemplate) {
-    return <StyledVisualStoryText element={element} dangerouslySetInnerHTML={{ __html: text }} />;
-  }
-  return <StyledText element={element} dangerouslySetInnerHTML={{ __html: text }} />;
+  return storyTemplate ? (
```

Comment:
can rename `storyTemplate` to `visualStoryTemplate`, can use lodash get to get the template